### DISCUSSION TITLE
Backup API

### DISCUSSION THREAD

**Discussion by powej2** [Art-of-WiFi/UniFi-API-client#209] [General]
Is there any method of using the generate backup api to download the created config?

**Comment by malle-pietje** [comment]
Not sure what you are trying to achieve. Can you elaborate?

**Reply by powej2** [reply]
I would like to run a script to download the automated UniFi backup config file to be able to save it on another server

**Comment by malle-pietje** [comment]
Okay, as the comments for the method say, the results contain the URL from where you can download the newly generated backup (e.g. using the curl functions in PHP). Try that and share any specific questions you may have if that doesn’t work.
